### ISSUE TITLE
CELA P1_A11y_code.visualstudio.com_VisualStudioCode_Common questions_AI4D: Label given for the button and the name property assigned are different.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/vscode-docs#3590]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#CELA P1_A11y_code.visualstudio.com;#A11ySev2;#A11yTCS;#A11yWCAG2.1;#Benchmark;#ChromiumEdge;#DesktopWeb;#MAS2.5.3;#Win10;#Visual Studio Code;#CELAP1_code;#AI4D;

**Environment Details:**
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com/docs/python/python-tutorial
Windows Version: Windows10
Browser version: Microsoft Edge Version 81.0.416.60 (Official build) beta (64-bit)

**Repro Steps:**
1)Launch the application url :
https://code.visualstudio.com/docs/editor/versioncontrol
2)Using Version Control in VS Code Page will get open.
3)Go through "Common questions" heading navigate to the feedback part and check for the label that is given to the button yes/no and the announcement made for the same buttons by screen reader.

**Actual:**
When we navigate to the yes/no buttons in the feedback section under "Common questions" heading these buttons have different text assigned than the label that is given to the buttons that is visually seen by users.

**Expected:**
The name for the buttons should be announced exactly the same as it is visually seen for all the users.

**User Impact:**
Screen reader users might get confused as the visually seen data and screen reader announcements are different.

**Attachment link for Reference:**
[3590_P1_A11y_code.visualstudio.com_VisualStudioCode_Common questions_AI4D.zip](https://github.com/microsoft/vscode-docs/files/4511888/3590_P1_A11y_code.visualstudio.com_VisualStudioCode_Common.questions_AI4D.zip)

**Comment by v-atkulk** [comment]
Going ahead and closing this issue as per the discussion with our SME.

**Comment by Ashutoshsahoo164** [comment]
GitHubTags:#Closed

**Comment by 12-shweta** [comment]
#A11yMAS;
